art/attacks/evasion/adversarial_patch/adversarial_patch_pytorch.py

```diff
@@ -96,7 +96,7 @@ def __init__(
         :param learning_rate: The learning rate of the optimization.
         :param max_iter: The number of optimization steps.
         :param batch_size: The size of the training batch.
-        :param patch_shape: The shape of the adversarial patch as a tuple of shape HWC (width, height, nb_channels).
+        :param patch_shape: The shape of the adversarial patch as a tuple of shape CHW (nb_channels, width, height).
```

Comment:
Following the tuple description, I assume you mean `CWH` right?

Reply:
It should be `CHW`, but the tuple needs to be updated. Fixed with 63155f1.